web/src/containers/viewOnly/AssurancesAndCompliance.js

```diff
@@ -1,43 +1,43 @@
 import PropTypes from 'prop-types';
-import React, { Fragment } from 'react';
+import React from 'react';
 import { connect } from 'react-redux';
 
-import { CheckCircle, TimesCircle } from '../../components/Icons';
 import regLinks from '../../data/assurancesAndCompliance.yaml';
 import { selectFederalCitations } from '../../reducers/apd.selectors';
 import { t } from '../../i18n';
+import {LinkOrText} from '../AssurancesAndCompliance'
 
 const AssurancesAndCompliance = ({ citations }) => {
   return (
     <div>
       <h2>Assurances and Compliance</h2>
-      {Object.entries(regLinks).map(([name]) => (
+      {Object.entries(regLinks).map(([name, regulations]) => (
         <div key={name} className="ds-u-margin-bottom--3">
           <h4 className="ds-h4">
             {t(`assurancesAndCompliance.headings.${name}`)}
           </h4>
           <ul className="ds-c-list--bare">
             {citations[name].map(({ title, checked, explanation }) => (
-              <li key={title}>
-                {checked === true && (
-                  <span style={{ fontSize: '20px', paddingRight: '8px' }}>
-                    <CheckCircle />
-                  </span>
-                )}
-                {checked === false && (
-                  <span style={{ fontSize: '20px', paddingRight: '8px' }}>
-                    <TimesCircle />
-                  </span>
-                )}
-                {checked === '' && <div className="fake-radio-button" />}
-                {title}
-                {checked === false && (
-                  <Fragment>
-                    <br />
-                    <strong>Explanation:</strong>{' '}
-                    {explanation || <em>No explanation given</em>}
-                  </Fragment>
-                )}
+              <li key={title} style={{"page-break-inside": "avoid"}} className="ds-u-margin--1">
+                <div className="ds-u-padding-bottom--1">
+                  Does this APD comply with <LinkOrText link={regulations[title]} title={title} />?
+                </div>
+                {checked && <div><strong>Yes </strong></div> }
+
+                {checked === false &&
+                  <div className="ds-l-container ds-u-margin-left--0">
```

Comment:
I think container only uses margin to center itself on the page. So instead of `ds-u-margin-left--0` you can do `ds-u-padding-x--0` to remove any padding.

Reply:
> I think container only uses margin to center itself on the page. So instead of `ds-u-margin-left--0` you can do `ds-u-padding-x--0` to remove any padding.

Good call.  I didn't realize how out of line they were.  I think everything should be riding the left rail now.